orion/sites/admin/src/announce/announceEdit.js

```diff
@@ -55,3 +153,8 @@ export class AnnounceEditPage extends React.Component {
     );
   }
 }
+
+function createDate() {
+  let now = new Date(Date.now());
+  return now;
```

Comment:
add a TODO. We are going to use a DateTime selector to figure out how to select a certain day and at what time to post an announcement.